src/index.html

```diff
@@ -1,17 +1,185 @@
+          <input type="text"
+          name="name"
+          autocomplete="off"
+           >
+        </label>
+        <div>
+          <label class="form-field how-old">
+            How old are you
+
+            <input
+              type="number"
+              id="tentacles"
+              name="tentacles"
+              min="1"
+              max="100"
+              value="12"
+            >
+          </label>
+        </div>
+        <div>
+          <label class="form-field date-birth">
+            Full date of birth:
+            <input
+              type="date"
+              id="start"
+              name="trip-start"
+              min="1922-01-03"
+              max="2022-01-03"
+            >
+          </label>
+        </div>
+
+        <div>
+          <label class="form-field">
+            <input
+            type="checkbox"
+            name="term"
+            >
+            I accept the term of the agreement
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="registration">
+        <legend>Registration</legend>
+        <label class="form-field email">
+          Email:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="form-field password">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="5"
+            maxlength="10"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="an-inter-fact">
+        <legend>An interesting fact about you!</legend>
+        <div>
+          <div>
+            <label class="form-field cats">
+              Do you love cats?
+              <input type="radio"
+              id="Yes"
+              name="cat"
+              value="yes"
+               >
+              - Yes
+              <input
+              type="radio"
+              id="No"
+              name="cat"
+              value="no"
+              >
+              - No
+            </label>
+          </div>
+          <div>
+            <label class="form-field color">
+              What is your favorite color?
+              <input
+              type="color"
+              id="head"
+              name="head"
+              value="#e66465"
+              >
+            </label>
+          </div>
+
+          <label class="form-field time-to-bed">
+            What time do you go to bed?
+            <input
+              type="time"
+              id="appt"
+              name="appt"
+              min="00:00:00"
+              max="23:59:59"
+              required
+            >
+          </label>
+
+          <div>
+            <label class="form-field brandOfcars">
+              What are you favorite brand of cars?
+              <select name="multiplei" required>
+                <option value=" "></option>
```

Comment:
No need to provide empty value="" attribute for inputs. Add it only if a certain value is specified from the start, or use placeholders.